sdk/python/feast/client.py

```diff
@@ -668,9 +674,13 @@ def get_online_features(
                 "feature_set:feature" where "feature_set" & "feature" refer to
                 the feature and feature set names respectively.
                 Only the feature name is required.
-            entity_rows: List of GetFeaturesRequest.EntityRow where each row
+            entity_rows:
+                List of GetFeaturesRequest.EntityRow where each row
                 contains entities. Timestamp should not be set for online
                 retrieval. All entity types within a feature
+                OR
+                List of Dict[str, Union[bool,bytes,float,int,str,List[bool,bytes,float,int,str]]]]
```

Comment:
This explanation is super confusing. Can we just explain this in a human readable way? 

A list of dictionaries where the key is an entity and the value is either a Python native type or feast.types.Value? You can remove all mention of EntityRow since we dont want people to use it.